src/components/collection/SummaryDetails.tsx

```diff
@@ -0,0 +1,226 @@
+import dynamic from 'next/dynamic';
+
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { Box, Chip, Grid, Skeleton, Typography, styled } from '@mui/material';
+
+import { LIBRARY } from '@graasp/translations';
+
+import { CATEGORY_COLORS, CATEGORY_TYPES } from '../../config/constants';
+import {
+  SUMMARY_CATEGORIES_CONTAINER_ID,
+  SUMMARY_CC_LICENSE_CONTAINER_ID,
+  SUMMARY_CC_LICENSE_NO_LICENSE_ID,
+  SUMMARY_CREATED_AT_CONTAINER_ID,
+  SUMMARY_LANGUAGES_CONTAINER_ID,
+  SUMMARY_LAST_UPDATE_CONTAINER_ID,
+} from '../../config/selectors';
+
+const { DateTime } = require('luxon');
+
+const { CreativeCommons } = {
+  CreativeCommons: dynamic(
+    () => import('@graasp/ui').then((mod) => mod.CreativeCommons),
+    { ssr: false },
+  ),
+};
+
+const DetailCard = styled(Box)(() => ({
+  border: '1px solid #ddd',
+  borderRadius: 7,
+  padding: 20,
+  height: '100%',
+}));
+
+type SummaryDetailsProps = {
+  createdAt: string;
+  lastUpdate: string;
+  lang: string;
+  languages: Immutable.List<{
+    name: string;
+  }>;
+  levels: Immutable.List<{
+    name: string;
+  }>;
+  disciplines: Immutable.List<{
+    name: string;
+  }>;
+  isLoading: boolean;
+  ccLicenseAdaption: string | undefined;
+};
+
+const SummaryDetails: React.FC<SummaryDetailsProps> = ({
+  isLoading,
+  createdAt,
+  lastUpdate,
+  lang,
+  languages,
+  levels,
+  disciplines,
+  ccLicenseAdaption,
+}) => {
+  const { t } = useTranslation();
+
+  const requireAccreditation = ccLicenseAdaption?.includes('BY');
+  const allowCommercialUse = !ccLicenseAdaption?.includes('NC');
+  const allowSharing = (() => {
+    if (ccLicenseAdaption?.includes('SA')) {
+      return 'alike';
+    }
+    return ccLicenseAdaption?.includes('ND') ? 'no' : 'yes';
+  })();
```

Comment:
How does this handle legacy data (the current CC license strings)?

Reply:
I've pushed a commit to handle the old licenses:
```typescript
  if (['alike', 'allow'].includes(ccLicenseAdaption)) {
    return {
      requireAccreditation: true,
      allowCommercialUse: true,
      allowSharing: ccLicenseAdaption === 'alike' ? 'alike' : 'yes',
    };
  }
```